app/views/admin/bulk_uploads/set_titles.html.erb

```diff
@@ -1,34 +1,65 @@
-<% page_title "Bulk upload for: #{@edition.title}" %>
+<% content_for :page_title, "Bulk upload for: #{@edition.title}" %>
+<% content_for :title, "Edit file attachment" %>
```

Comment:
Is this the right title? This page is for multiple attachments, so I imagine it should at least be:

```suggestion
<% content_for :title, "Edit file attachments" %>
```

It's worth noting the Bootstrap design uses the title "Set titles for your attachments"